cli/src/utils.rs

```diff
@@ -108,6 +109,87 @@ where
     Ok(())
 }
 
+pub async fn load_client_and_run<F, Fut>(
```

Comment:
This function is very similar to `load_device_and_run`, can it be possible to shared identical code between the two ?

Reply:
Done